specs/eip4844/p2p-interface.md

```diff
@@ -58,6 +58,14 @@ class SignedBlobsSidecar(Container):
     signature: BLSSignature
 ```
 
+### `SignedBeaconBlockAndBlobsSidecar`
+
+```python
+class SignedBeaconBlockAndBlobsSidecar(Container):
+    beacon_block: SignedBeaconBlock
+    blobs_sidecar: SignedBlobsSidecar
```

Comment:
Verifying the sidecar signature helps ensure the KZG commitments are worth verifying, but would it really result in DoS / performance issues when anyone changes it to invalid data? Technically we compare the sidecar against the KZG commitments in the beacon block already, so we do not have to sign the sidecar.

Reply:
Yeah I don't think we need to sign the side car. This also informs whether or not we send signed or unsigned side cars in blobs by range/root request.

Reply:
Yeah, I agree that comparing KZG commitments in the beacon block is sufficient. We don't need the sidecar signature

@realbigsean range/root request will be decoupled and it's unsigned there